capstone/src/main/java/com/google/sps/servlets/VoterServlet.java

```diff
@@ -35,11 +38,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     URLConnection connection = new URL(url + "?" + query).openConnection();
     connection.setRequestProperty("Accept-Charset", charset);
     InputStream result = connection.getInputStream();
+    Scanner scanner = new Scanner(result);
```

Comment:
Is any of this stuff we can move to helper methods? It would be easier to test said helper methods than to try and test this big one with an API call built in

Reply:
+1